wagtail/admin/tests/test_workflows.py

```diff
@@ -1535,6 +1538,8 @@ def test_get(self):
         self.assertTemplateUsed(response, "wagtailadmin/workflows/task_chooser/includes/results.html")
         self.assertTemplateNotUsed(response, "wagtailadmin/workflows/task_chooser/includes/create_form.html")
         self.assertFalse(response.context['searchform'].is_searching())
+        # check that only active (non-disabled) tasks are listed
+        self.assertEqual([_.name for _ in response.context['tasks'].object_list], ['Enabled foo', 'Moderators approval'])
```

Comment:
I find it confusing to name this variable `_` since that makes me expect it will be ignored. Would it be clearer if we called it `task`?

Reply:
yep can do, bad habit from my Javascript life haha